docs/source/models/causes/neonatal/index.rst

```diff
@@ -158,32 +174,62 @@ unit time.
     * - Symbol
       - Name
       - Definition
-    * - mr
-      - mortality risk
-      - The probability that a simulant who was born alive dies during the neonatal period
+    * - enn_mr
+      - mortality risk during the early neonatal period
+      - The probability that a simulant who was born alive dies during the first 7 days
+    * - lnn_mr
+      - mortality risk during the late neonatal period
+      - The probability that a simulant who was born alive dies between day 8 to 28 of life
 
 
 Data Tables
```

Comment:
As I just noted on the preterm page, I think the "Data Tables" section should be reserved for the actual data tables, and we should add a new section (e.g., "Modeling strategy") to contain the modeling strategy. Also, it would be nice to add a table of contents to all the pages to make them easier to navigate.